packages/itwinui-css/src/button/active.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+
+@mixin iui-button-active {
+  --_iui-button-active-stripe-inset-block: initial;
+  --_iui-button-active-stripe-inset-inline: initial;
+  --_iui-button-active-stripe-background: var(--iui-color-border-accent);
+  --_iui-button-icon-fill: var(--iui-color-icon-accent);
+  --_iui-button-icon-fill-hover: var(--iui-color-icon-accent);
+  --_iui-field-color-background-disabled: var(--iui-color-background-disabled);
```

Comment:
Is this variable needed to be set here? Because it seems to be set to the default value from `iui-field`.

Reply:
It does appear to be necessary.

With that line included:
![Screenshot 2024-04-02 at 5 39 34 PM](https://github.com/iTwin/iTwinUI/assets/849817/aa29814c-0bb6-4d39-9bf6-d277d5d0a08a)

After removing that line:
![Screenshot 2024-04-02 at 5 40 14 PM](https://github.com/iTwin/iTwinUI/assets/849817/ca4323fd-75a3-474a-ac83-413ba5fc0bae)

I think it must have to do with removing the background from the disabled borderless button.